cli.js

```diff
@@ -30,6 +31,7 @@ var cli = meow({
 }, {
   alias: {
     i: 'infile',
+    h: 'help',
     o: 'outfile',
     w: 'overwrite',
```

Comment:
Here we thought `--overwrite` was a bit confusing, I changed it to `--same-file`